setup.py

```diff
@@ -156,6 +156,30 @@
     ]})
 setup_kwargs['entry_points'] = entry_points
 
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Education',
+    'Intended Audience :: End Users/Desktop',
+    'Intended Audience :: Science/Research',
+    'License :: DFSG approved',
+    'License :: OSI Approved :: MIT License',
+    'Natural Language :: English',
+    'Operating System :: POSIX',
```

Comment:
I wonder if that is anyhow used by PyPi/pip to decide/limit installations on only POSIX compliant systems... If we list it I think we should then add `MacOS :: MacOS X` and `Microsoft :: Windows` at least. and may be even `Unix`?

Reply:
MacOSX is a POSIX OS. I intentionally chose that to say "in principle agnostic, but...". I did not want to include/advertise Windows, because we are not there yet.

Reply:
I just looked at the hierarchy in the side bar on https://pypi.org/search/?c=Operating+System+%3A%3A+POSIX+%3A%3A+GNU+Hurd where OSX has its own section. I am ok to no Ad for Windows (yet)

Reply:
Indeed. Searching for MacOS does not give any hits for packages that are 'POSIX' alone. So we need to add that. Thx for checking.